visualize-admin/visualization-tool
# fix: Compact table view

Fixes #1752

This PR improves the `switch` statement in compact mobile cell to not access uninitialized variable + improves the style of the compact view, so that the rows do not overlap with each other.

## How to reproduce
1. Go to [this link](https://test.visualize.admin.ch/en/v/JApGFtU6sNLO?dataSource=Prod).
2. Switch to compact table layout.
3. ❌ See that it breaks the chart.

## How to test
1. Go to [this link](https://visualization-tool-git-fix-compact-table-view-ixt1.vercel.app/en/v/AYuDcdSVul7A?dataSource=Prod).
2. Switch to compact table layout.
3. ✅ See that it works and that the rows are not overlapping.
